test/errors/not-an-article.test.ts

```diff
@@ -0,0 +1,21 @@
+import NotAnArticle from '../../src/errors/not-an-article';
+
+describe('not an article error', (): void => {
+  it('should be an error', async (): Promise<void> => {
+    const error = new NotAnArticle();
+
+    expect(error).toBeInstanceOf(Error);
+  });
+
+  it('should have a message', async (): Promise<void> => {
+    const error = new NotAnArticle();
+
+    expect(error.message).toBe('Article type must be http://schema.org/Article (none was given)');
+  });
+
+  it('may have types', async (): Promise<void> => {
+    const error = new NotAnArticle(['http://schema.org/NewsArticle', 'http://schema.org/Book']);
+
+    expect(error.message).toBe('Article type must be http://schema.org/Article (\'http://schema.org/NewsArticle, http://schema.org/Book\' was given)');
```

Comment:
We should make the types accessible and test here.

Reply:
> We should make the types accessible and test here.

Can you give more detail here? Not sure what you mean, sorry.

Reply:
Meant add a `types` property on the error (like `id` on `ArticleNotFound`).